index.html

```diff
@@ -143,6 +148,67 @@
                 padding: 10px 10px;
             }
             }
+        .why-us {
+            background-color: var(--darkblue);
+            padding: 100px 195px 100px 195px;
+        }
+        .why-choose-us {
+            display: flex;
+            flex-direction: column;
+            text-align: center;
+        }
+        .grid-content {
+            display: grid;
+            grid-template-columns: repeat(3,328px);
+            background-color: var(--darkblue);
+            justify-content: center;
+        }
+        .grid-item {
+            background-color: white;
+            border: 15px solid var(--darkblue);
+            text-align: center;
+            color: #737373;
+        }
+        .grid-item-header {
+            display: flex;
+            gap: 15px;
+            justify-content: center;
+            padding: 35px 35px 0 35px;
```

Comment:
Just curious, are these the right padding dimensions? what is the purpose of 0?

Reply:
I didn't put 35px on the bottom cause the item has the gap of 15px.
![image](https://github.com/Irrreny/Project_KMarketer/assets/141719439/25c8b597-aa93-44af-9500-92688783593a)

Reply:
well, if i remove the padding, it looks like this :)
![image](https://github.com/Irrreny/Project_KMarketer/assets/141719439/9367c408-c03a-450b-9c84-5b483e3afdfc)

So, i just saw how it looks and kept the padding :)

Reply:
Hmm, I see you didn't add padding to your "grid-item". Never mind then, I thought it was in both parts :)

Reply:
Emma is right. The padding should be set to the parent. So your grid-item would have `padding: 35px 40px;`
children of grid-item should not have paddings. It should work.